sdk/options.go

```diff
@@ -16,6 +16,8 @@ type Option func(*config)
 
 // Internal config struct for building SDK options.
 type config struct {
+	// Platform configuration structure is subject to change. Consume via accessor methods.
+	PlatformConfiguration   PlatformConfiguration
```

Comment:
If this is the case, should we lowercase `platformConfiguration`?

Reply:
```suggestion
	platformConfiguration   PlatformConfiguration
```

Reply:
This was from feedback from @strantalis to put these accessors a level beyond the top-level SDK methods. Exposing the `PlatformConfiguration` means the below:

```golang
// if exposed:
client := sdk.New()
// client.PlatformConfiguration is map[string]interface{}
client.PlatformConfiguration.Issuer() // gives the well-known issuer and same APIs for well-known config accessors

// if unexposed
client := sdk.New()
client.PlatformIssuer() // a new top-level method for every well-known accessor
```